src/nidcpower/metadata/attributes.py

```diff
@@ -11,7 +11,7 @@
         'type': 'ViBoolean'
     },
     1050005: {
-        'access': 'read-write',
+        'access': 'read only',
```

Comment:
Comment so I can review.

Reply:
This PR was just to test the system tests trigger with the new nimibot, no actual changes. I'll close the PR.